.github/workflows/publish-draft-release.yml

```diff
@@ -3,8 +3,8 @@ name: Release - Publish draft
 on:
   push:
     tags:
-      # Catches v1.2.3 and v1.2.3-rc1
-      - v[0-9]+.[0-9]+.[0-9]+*
+      # Catches v1.2.3 and v1.2.3
```

Comment:
Now the comment's second example is redundant as it is the same as the first (`v1.2.3`).
Could you please extend the comment here to incorporate the fact that it catches SDK **and** Docker releases?